### ISSUE TITLE
Fan control not working

### ISSUE THREAD

**Issue by AmshaSharam** [supermerill/SuperSlicer#4009]
### Version
Version 2.5.59.3

### Operating system type + version
_windows 10

### Behavior
*The fan control windows settings do not work, if i disable or sert a value less than the default ,it ignores.
![fan](https://github.com/supermerill/SuperSlicer/assets/97839782/c9eb4535-e209-4ed0-a5ce-19c1defe1859)
![settings](https://github.com/supermerill/SuperSlicer/assets/97839782/05e54c45-b917-4e3d-a180-752302a682d7)

**Comment by supermerill** [comment]
Yes, that is the behaviour. the default fan act as a min speed. I guess i can spit it in two now.
- renaming back the "default speed" to "min speed"
- removing the old "fan_always_on" and adding the real default speed.

**Comment by AmshaSharam** [comment]
Thanks
